CHANGES.rst

```diff
@@ -22,6 +22,9 @@ API Changes
 - Viewers now can access the calling Jdaviz application using
   ``viewer.session.jdaviz_app``. [#1051]
 
+- Viewers now can access the Jdaviz application via ``viewer._jdaviz_app`` and the
```

Comment:
`viewer._jdaviz_app` duplicates `viewer.session.jdaviz_app` from #1051. They should be consolidated.

In general, I am against prepending `_` to these new attributes because that implies they are hidden, but maybe that is okay if that means they are hidden only for users and not developers? And if they are supposed to be hidden for users, then maybe we don't even need change log (so we will have to remove this entry *and* the entry from #1051)?